data_model/src/validator.rs

```diff
@@ -94,6 +94,16 @@ pub mod model {
     }
 }
 
+impl Validator {
+    /// Create empty mock validator to use as stub in tests
+    // TODO: consider feature like `mock` to provide convenience methods for tests
```

Comment:
You think this is needed for unit tests? For integration tests it's not needed definitely

Reply:
Yes, i was thinking about unit tests in genesis crate. Unfortunately ‘#[cfg(test)]‘ doesn't work across crates.

Reply:
it should be possible to write small inline wasm validator that always passes for unit tests. No need for feature in that case

Reply:
1. We decided to remove `mock` from as many crates as possible. This is going backwards. 
2. it's better to have different samples as files than it is to have them be generated for mocking purposes only. 
3. As @mversic has said on several previous occasions, if a state is unrepresentable, tests that work with unrepresentable state are meaningless.

Reply:
I will remove this method since i observed that we don't really need it.